src/main/java/ti4/listeners/annotations/AnnotationHandler.java

```diff
@@ -150,19 +151,22 @@ private static <T extends ListenerContext> Consumer<T> buildConsumer(Method meth
                 context.setShouldSave(save);
                 method.invoke(null, args.toArray());
             } catch (InvocationTargetException e) {
-                BotLogger.error("Error within handler \"" + method.getDeclaringClass().getSimpleName() + "#" + method.getName() + "\":", e.getCause());
+                LogMessageOrigin origin = null;
                 for (Object arg : args) {
                     if (arg instanceof ButtonInteractionEvent buttonInteractionEvent) {
+                        origin = new LogMessageOrigin((GenericInteractionCreateEvent) arg);
                         buttonInteractionEvent.getInteraction().getMessage()
                             .reply("The button failed. An exception has been logged for the developers.")
                             .queue();
                     }
                     if (arg instanceof StringSelectInteractionEvent selectInteractionEvent) {
+                        origin = new LogMessageOrigin((GenericInteractionCreateEvent) arg);
                         selectInteractionEvent.getInteraction().getMessage()
                             .reply("The selection failed. An exception has been logged for the developers.")
                             .queue();
                     }
                 }
+                BotLogger.error(origin, "Error within handler \"" + method.getDeclaringClass().getSimpleName() + "#" + method.getName() + "\":", e.getCause());
```

Comment:
There were several errors I couldn't really track down because they were a call stack, but no game context. This will try to get the game context for button errors every time.
- example: https://discord.com/channels/943410040369479690/1341438476528123904/1376507434323148830
- example: https://discord.com/channels/943410040369479690/1341438476528123904/1376239752226799661